packages/graph/hash_graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -38,6 +38,71 @@ impl QueryRecord for EntityType {
     type Path<'q> = EntityTypeQueryPath;
 }
 
+impl RecordPath for EntityTypeQueryPath {
+    fn expected_type(&self) -> ParameterField {
+        match self {
+            Self::OwnedById | Self::CreatedById | Self::UpdatedById => ParameterField {
+                parameter_type: ParameterType::Uuid,
+                optional: false,
+            },
+            Self::RemovedById => ParameterField {
+                parameter_type: ParameterType::Uuid,
+                optional: true,
+            },
+            Self::BaseUri => ParameterField {
+                parameter_type: ParameterType::BaseUri,
+                optional: false,
+            },
+            Self::VersionedUri => ParameterField {
+                parameter_type: ParameterType::VersionedUri,
+                optional: false,
+            },
+            Self::Version => ParameterField {
+                parameter_type: ParameterType::UnsignedInteger,
+                optional: false,
+            },
+            Self::Title => ParameterField {
+                parameter_type: ParameterType::Text,
+                optional: false,
+            },
+            Self::Description => ParameterField {
+                parameter_type: ParameterType::Text,
+                optional: true,
+            },
+            Self::Default | Self::Examples | Self::Required | Self::RequiredLinks => {
+                ParameterField {
+                    parameter_type: ParameterType::Any,
```

Comment:
Why are these all `Any`? Do we treat arrays of strings as `Any`?

Reply:
I have not added a `ParameterType::List` yet (we probably want a function like "contains" in the future so we can query for entity types with specific required fields), so I just put `Any` as a placeholder. For `Default` or `Examples` this could be anything, which is allowed by the schema so it's impossible to check without investigating the entity type before (which is now known at this point).